### DISCUSSION TITLE
Disable embedding entirely

### DISCUSSION THREAD

**Discussion by paulhargreaves** [advplyr/audiobookshelf#3206] [Q&A] [answered]
Is there a setting I can set to disable the embed functionality entirely?

**Comment by advplyr** [comment]
I don't see what the use-case would be for that. You have other admins on your server that you don't want to click the embed button?

**Comment by paulhargreaves** [comment]
There are open tickets for corruption caused by embedding. I don't ever want to accidentally change my files so I want to make sure that even if I click on the wrong thing by accident then it won't attempt to do updates.

**Reply by nichwall** [reply, marked as answer]
You can make your media files read only so ABS can't touch them at all. If you're using docker, you can just make the mount read only (but you will not be able to store metadata or covers edited by ABS with the items).

This corruption was due to tone which was replaced with ffmpeg in 2.11.0, but making backups before editing files is always a good idea (with ABS or otherwise).

**Reply by paulhargreaves** [reply]
I'll go with the mount ro option - I'm fine with all the metadata being stored in the metadata mount.
